resource/api_model_test.go

```diff
@@ -0,0 +1,549 @@
+//nolint:dupl
```

Comment:
I've had to do this for duplicated code in tests, too. I don't know why, but you have to put it at the top of the file 😆 .